src/components/rounds-list/rounds-list.tsx

```diff
@@ -0,0 +1,24 @@
+import { Round } from "@/app/events/[event]/participants/page";
+import { ParticipantsList } from "../participants-list";
```

Comment:
Avoid relative imports, use the `@/` prefix to import.

Reply:
Fixed